Vic2ToHoI4/Source/Hoi4Outputter/Diplomacy/OutAiPeaces.cpp

```diff
@@ -20,7 +20,7 @@ void HoI4::outAiPeaces(
 				std::ofstream outFile(
 					"output/" + theConfiguration.getOutputName() + "/common/ai_peace/1_communist.txt"
 				);
-				if (majorIdeologies.count("communist") > 0)
+				if (majorIdeologies.count("communism") > 0)
```

Comment:
If you want a bool 
if (majorIdeologies.count("communism")){ }  
is sufficient